internal/controller/velero_test.go

```
@@ -21,6 +21,10 @@ import (
 	"github.com/google/go-cmp/cmp"
 	"github.com/onsi/ginkgo/v2"
 	"github.com/onsi/gomega"
+	oadpv1alpha1 "github.com/openshift/oadp-operator/api/v1alpha1"
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gci)